jiant/modules/simple_modules.py

```diff
@@ -94,6 +94,30 @@ def from_params(cls, d_inp, n_classes, params):
         )
 
 
+class SOPClassifier(nn.Module):
+    """
+    Task head for sentence order prediction task. We approximate the pooled output from ALBERT
```

Comment:
approximate => implement 
The data and checkpoint loaders aren't quite correct, but this is the exact right model, no?